benchmark/parse_stream.cpp

```diff
@@ -71,7 +71,7 @@ int main (int argc, char *argv[]){
         std::wclog << "parse_many: Speed per batch_size... from " << MIN_BATCH_SIZE
                 << " bytes to " << MAX_BATCH_SIZE << " bytes..." << std::endl;
         std::cout << "Batch Size\t" << "Gigabytes/second\t" << "Nb of documents parsed" << std::endl;
-        for (size_t i = MIN_BATCH_SIZE; i <= MAX_BATCH_SIZE; i += (MAX_BATCH_SIZE - MIN_BATCH_SIZE) / 50) {
+        for (size_t i = MIN_BATCH_SIZE; i <= MAX_BATCH_SIZE; i += (MAX_BATCH_SIZE - MIN_BATCH_SIZE) / 100) {
```

Comment:
One wonders if a binary search is what we're really after ...

Reply:
The idea was to be able to see the whole curve and know that it is modal downward concave curve.

Reply:
We can change it for sure, but if you throw in some noise, it is going to be a fun algorithm to watch.

Reply:
Good point. One bad result means we go left when we shoulda gone right.

Reply:
This whole thing should probably just go away. It is ugly to ask the user to tweak the batch size. That is part of the motivation behind this PR: we would want to get good performance no matter what the batch size is, up to some reasonable limits.